packages/kit-headless/src/components/progress/progress-indicator.tsx

```diff
@@ -0,0 +1,24 @@
+import { PropsOf, component$, useContext } from '@builder.io/qwik';
+import { getProgressState } from './util';
+import { ProgressContext } from './progress-context';
+
+type ProgressIndicatorElement = PropsOf<'div'>;
+export const Indicator = component$<ProgressIndicatorElement>((props) => {
+  const { ...indicatorProps } = props;
+
+  const { max, value } = useContext(ProgressContext);
+  const translateX = value ? `translateX(-${100 - value}%)` : undefined;
+
+  return (
+    <div
+      style={{
+        transform: translateX,
+        transition: 'transform 0.3s',
+      }}
+      data-state={getProgressState(value, max)}
```

Comment:
Another thing, we are moving away from data-state, and more towards something like data-open or data-closed, this is much more intuitive to style. 

I would take a look at how this is done in the select.

Also the headless kit should have as minimal styles as possible. Most components adhere to that principle, with the outlier being the carousel (which we need to change)

Reply:
`data-state` can be `data-progress` (used in other libraries, but not all.)

what do you think?

Reply:
> `data-state` can be `data-progress` (used in other libraries, but not all.)
> 
> what do you think?

Sounds good to me 😄, and if we get some feedback on the name from consumers we can always make some changes, we are currently in a phase of qwik ui where we are trying to "standardize" the components. The select component being the first standard component in the library (conventions, api, naming, etc.)

Reply:
Awesome, done.